diffsims/crystallography/get_hkl.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017-2023 The diffsims developers
```

Comment:
I think you might want to set this as 2024 or are you going to automate the entire package?

Reply:
I suggest to automate it with licenseheaders in a final PR prepping the `main` branch for a release. This file was renamed from *reciprocal_lattice_point.py*, the header left unchanged.

Reply:
Works for me!